src/pysdmx/api/gds/__init__.py

```diff
@@ -202,8 +196,7 @@ def get_sdmx_apis(
         """Get the list of SDMX API versions.
 
         Args:
-            api_version: The ID of the SDMX API version
-            to query. Defaults to '*'.
+            api_version: The SDMX_API to query. Defaults to '*'.
```

Comment:
Should we say something like: "The version of the SDMX API to be returned." instead of "The SDMX_API to query."?